File: Flower.Data/Entities/Page.cs
using System;
using System.ComponentModel.DataAnnotations;

namespace Flower.Data.Entities
{
    public class Page
    {
        [Key]
        public int Id { get; set; }

        [Required]
        [MaxLength(500)]
        public string Title { get; set; } = string.Empty;

        [MaxLength(300)]
        public string? Slug { get; set; }

        [Required]
        public string Content { get; set; } = string.Empty;

        public bool IsActive { get; set; } = true;

        public DateTime CreatedAt { get; set; } = DateTime.UtcNow;

        public DateTime? UpdatedAt { get; set; }
    }
}

File: flower-admin.frontend/src/types/page.ts
export interface PageDTO {
  id: number
  title: string
  slug?: string
  content: string
  isActive: boolean
  createdAt: string
  updatedAt?: string
}

export interface CreatePageDTO {
  title: string
  slug?: string
  content: string
  isActive?: boolean
}

export interface UpdatePageDTO {
  id: number
  title: string
  slug?: string
  content: string
  isActive?: boolean
}
